### ISSUE TITLE
Wrong darwin/x64 version (3.3.3 - should be 3.4.2)

### ISSUE THREAD

**Issue by borestad** [eugeneware/ffmpeg-static#17]
**Version of ffmpeg-static**
2.2.0 

**Version of ffmpeg**
3.3.3

**Verified manually**
Extract https://registry.yarnpkg.com/ffmpeg-static/-/ffmpeg-static-2.2.0.tgz

```
./node_modules/ffmpeg-static/bin/darwin/x64/ffmpeg -version

ffmpeg version 3.3.3-tessus Copyright (c) 2000-2017 the FFmpeg developers
built with Apple LLVM version 8.0.0 (clang-800.0.42.1)
configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libass --enable-libbluray --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzvbi --enable-version3 --disable-ffplay --disable-indev=qtkit
libavutil      55. 58.100 / 55. 58.100
libavcodec     57. 89.100 / 57. 89.100
libavformat    57. 71.100 / 57. 71.100
libavdevice    57.  6.100 / 57.  6.100
libavfilter     6. 82.100 /  6. 82.100
libswscale      4.  6.100 /  4.  6.100
libswresample   2.  7.100 /  2.  7.100
libpostproc    54.  5.100 / 54.  5.100
```

**Comment by borestad** [comment]
Will need to verify other platforms as well

**Comment by derhuerst** [comment]
I've published `ffmpeg-static@2.2.1` with ffmpeg `3.4.2`, confirmed with darwin/x64 and linux/x64. Someone should check the others too though.
